xcode/Subconscious/Shared/Models/LoadingState.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  NetworkResourceState.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 4/6/23.
+//
+
+import Foundation
+
+enum LoadingState: Hashable {
```

Comment:
General enum for representing loading states. Useful in cases where you want a flag on a struct.

Reply:
I expect we'll expand this to include other failure cases in the future? No reason to add them until they come up though.